src/clostache/parser.clj

```diff
@@ -328,7 +327,7 @@
   [section data partials]
   (let [section-data ((keyword (:name section)) data)]
     (if (:inverted section)
-      (if (or (and (seqable? section-data) (empty? section-data))
+      (if (or (and (clojure.core/seqable? section-data) (empty? section-data))
```

Comment:
Since we aren't using incubator anymore can we drop the `clojure.core/` and remove the `refer-clojure`?

Reply:
done